Realm.Shared/RealmObject.cs

```diff
@@ -698,7 +701,8 @@ protected void SetNullableDateTimeOffsetValue(string propertyName, DateTimeOffse
                 NativeTable.set_null(_metadata.Table, _metadata.ColumnIndices[propertyName], (IntPtr)rowIndex);
         }
 
-        // TODO make not generic
+
+        // This is a generic fallback, we normally handle with woven type-specific setters above
```

Comment:
We do? This is for `RealmObject`, I don't think there is anything else for those?